Atomic/scan.py

```diff
@@ -414,15 +414,15 @@ def print_scan_list(self):
 
     def mount(self, mountpoint, image):
         m = mount.Mount()
-        m.set_args(self.args)
+        m.args = self.args
         m.mountpoint = mountpoint
         m.image = image
         m.shared = True
         m.mount()
 
     def unmount(self, mountpoint):
         m = mount.Mount()
-        m.set_args(self.args)
+        m.args = self.args
         m.mountpoint = mountpoint
```

Comment:
Instead of set_args which was causing problems based on type difference within mount's function since that checks for certain terms within the arguments, in the case of scan, the arguments simply only need to be initialized but they are not used.